scalafix-core/shared/src/main/scala/scalafix/internal/rule/LeakingImplicitClassVal.scala

```diff
@@ -22,7 +22,7 @@ case object LeakingImplicitClassVal extends Rule("LeakingImplicitClassVal") {
         val optPatch = for {
           anchorMod <- pMods.find(!_.is[Mod.Annot])
           if !pMods.exists(m => m.is[Mod.Private] || m.is[Mod.Protected])
-        } yield ctx.addLeft(anchorMod, "private ")
+        } yield ctx.addLeft(anchorMod, "private ").atomic
         optPatch.asPatch
 
     }.asPatch
```

Comment:
could you add a test for this? see https://github.com/scalacenter/scalafix/blob/master/scalafix-tests/input/src/main/scala/test/MissingFinalRewrite.scala#L14 and https://github.com/scalacenter/scalafix/blob/master/scalafix-tests/output/src/main/scala/test/MissingFinalRewrite.scala#L11 as an example

Reply:
I think we need to make similar change to `DottyVarArgPattern` and `DottyVolatileLazyVal`. Could you double check?